dev-libs/libmelf/libmelf-0.4.0-r1.ebuild

```diff
@@ -36,5 +34,5 @@ src_install() {
 	insinto /usr/include
```

Comment:
the `into /usr` can probably be nixed

Reply:
Also, hide the `dolib.a` behind a `static-libs` USE flag please

Reply:
static-libs USE flag added

Reply:
space and no tab after `&&`

Reply:
tab replaced with space